addons/account/views/account_invoice_workflow.xml

```diff
@@ -1,113 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-    <data>
-
-        <record id="wkf" model="workflow">
-            <field name="name">account.invoice.basic</field>
-            <field name="osv">account.invoice</field>
-            <field name="on_create">True</field>
-        </record>
-
-        <record id="act_draft" model="workflow.activity">
-            <field name="wkf_id" ref="wkf"/>
-            <field name="flow_start">True</field>
-            <field name="name">draft</field>
-        </record>
-        <record id="act_proforma2" model="workflow.activity">
-            <field name="wkf_id" ref="wkf"/>
-            <field name="name">proforma2</field>
-            <field name="action">write({'state':'proforma2'})</field>
-            <field name="kind">function</field>
-        </record>
-        <record id="act_open" model="workflow.activity">
-            <field name="wkf_id" ref="wkf"/>
-            <field name="name">open</field>
-            <field name="action">action_date_assign()
-action_move_create()
-invoice_validate()</field>
-            <field name="kind">function</field>
-        </record>
-        <record model="workflow.activity" id="act_open_test">
-            <field name="wkf_id" ref="wkf"/>
-            <field name="name">re-open</field>
-            <field name="action">write({'state':'open'})</field>
-            <field name="kind">function</field>
-        </record>
-        <record id="act_paid" model="workflow.activity">
```

Comment:
I am not sure the workflow is correctly converted. You should probably not have to manually call action_paid, because I think the workflow should handle it himself. Could you maybe check with accouting team ?